ote/src/cljs/ote/style/base.cljs

```diff
@@ -240,6 +240,26 @@
    :padding-right "0"
    :overflow-wrap "break-word"})
 
+(def new-change-indicator
+  {:height "6px"
```

Comment:
Should unit format be relative to font height like `rem`?
In extreme case font could be big and icon small.